usercp.php

```diff
@@ -10,6 +10,7 @@
 
 define("IN_MYBB", 1);
 define('THIS_SCRIPT', 'usercp.php');
+define("ALLOWABLE_PAGE", "removesubscription,removesubscriptions");
```

Comment:
This likely blocks every other connection to usercp.php. Please remove it, thank you.

Reply:
Isn't it for allow users even when they are not allowed to view board? I thought it was there to remove subscription from a mail link or something ....

Anyway, removed.

Reply:
@Shade- @euantorano or anyone can you guys please confirm this? As much I have cross-checked "ALLOWABLE_PAGE" is not blocking anything... Further cross-checking made me believe again that as I said it allows user to access page while viewing the board is not allowed.

Reply:
Yep, sorry, my fault. You can include that again.

Reply:
Added back. Thanks.